backend/app/services/llm/chain_executor.py

```
@@ -0,0 +1,344 @@
+    if config_blob.prompt_template:
+        return config_blob.prompt_template.template.replace("{{input}}", text)
+    return text
+
+
+def execute_chain_job(
+    request_data: dict,
+    project_id: int,
+    organization_id: int,
+    job_id: str,
+    task_id: str,
+    task_instance,
```

Ruff (0.14.14)

[warning] Unused function argument: `task_instance`

(ARG001)